code/game/objects/items/control_wand.dm

```diff
@@ -29,6 +29,13 @@
 	to_chat(user, "Now in mode: [mode].")
 
 /obj/item/weapon/door_remote/afterattack(obj/machinery/door/airlock/D, mob/user)
+	if(istype(D)) //yes, the istype has to be here, otherwise it runtimes if you misclick
+		if(D.is_special)
+			if(!(D.check_access(src.ID)))
```

Comment:
I don't understand why this is all necessary. If the door is special and doesn't work with the remote, why do you need to do an access check with it?